apps/rush-lib/src/cli/actions/PurgeAction.ts

```diff
@@ -44,7 +44,7 @@ export class PurgeAction extends BaseRushAction {
       const unlinkManager: UnlinkManager = new UnlinkManager(this.rushConfiguration);
       const purgeManager: PurgeManager = new PurgeManager(this.rushConfiguration, this.rushGlobalFolder);
 
-      unlinkManager.unlink();
+      unlinkManager.unlink(/*force:*/ true);
```

Comment:
What is this changing?

Reply:
Unlink is called by `rush purge`. This is only done to avoid the blocking error message which is intended for when calling `rush unlink`. I thought about putting the warning in the action class, but putting it in the utility felt more correct because of how we're blocking linking/unlinking. Though this can definitely be moved up. Same goes for the error message for `rush link`